### ISSUE TITLE
Problem with VMC

### ISSUE THREAD

**Issue by J013k** [aldostools/IRISMAN#12]
I have a problem with VMC.

I have selected (assigned) VMC for PS1 games:
![https://i.postimg.cc/R0Gtzv05/1.png](https://i.postimg.cc/R0Gtzv05/1.png)

When I load e.g. TR I'm getting this warring:
![https://i.postimg.cc/Pf027Qcb/2.png](https://i.postimg.cc/Pf027Qcb/2.png)

When I want to assign MC in-game it will crash\freeze:
![https://i.postimg.cc/fbF9bBYB/3.png](https://i.postimg.cc/fbF9bBYB/3.png)
![https://i.postimg.cc/wM5y0zZG/4.png](https://i.postimg.cc/wM5y0zZG/4.png)

I do not have this problem e.g. with ManaGunZ v1.38.

**Comment by Markamiga500** [comment]
I never tested ManaGunz v1.38

Are you able to test PAL PS1 games ?
are you able to actually create save files on PAL (F) games ?

Easy to create save files of NTSC PSX games on a NTSC console , but im still hurting my head concerning PAL PS1 games...

**Comment by J013k** [comment]
I'm able to create save files for PAL games on PAL console.

**Comment by Markamiga500** [comment]
Yes , but are you able to create save files using webman mod for NTSC games on a PAL console ?
thats THE question ;)

**Comment by J013k** [comment]
Can you explain me what "**create save files using webman mod**" means?

Or can you give me some photos, because I don't understand.

PAL save on PAL console (Tomb Raider [SLES_000.24]):
http://www.mediafire.com/file/rkai6k2u6pn0cgg/Tomb_Raider_PAL_save.7z.
NTSC save on PAL console (Oddworld - Abe's Oddysee [SLUS_001.90]):
http://www.mediafire.com/file/tiwtg6vnbbhr74v/Oddworld_-_Abe%2527s_Oddysee_NTSC_save.7z.

**Comment by Markamiga500** [comment]
sorry

simply meaned : AFTER mounting the game using webman and launching them
(PAL iso) on game i press PS button on PS3 controller to create save file
BUT for an unknow reason , everything deeply freeze black screen when i try
to create save file (only PAL games , NTSC work flawlessly)

Are you able to create save files of NTSC iso on your pal cfw ps3 ??

**Comment by J013k** [comment]
After mounting PS1 game, launch it through webMAN MOD and pressing PS button on PS3 controller,
I have only this menu:
![https://i.postimg.cc/QtQQp7QK/mc1.png](https://i.postimg.cc/QtQQp7QK/mc1.png)

**There is no option to create save file.**

Can you give me some photos, because I still don't understand with what you have problem?
Translated using  [DeepL](https://www.deepl.com/translator):
**Pouvez-vous me donner quelques photos, car je ne comprends toujours pas ce qui vous pose problème?**

I can create VMC (Virtual Memory Card) at XMB:
 Game -> Memory Card Utility -> Create New Internal Memory Card -> Internal Memory Card (PS)...
After that I can assign slot for it (e.g. Slot 1).

While playing game (NTSC or PAL) I can use save option embedded in game.
E.g. Oddworld - Abe's Oddysee [SLUS_001.90]:
![https://i.postimg.cc/QNSnrLsN/save-1-NTSC.png](https://i.postimg.cc/QNSnrLsN/save-1-NTSC.png)
![https://i.postimg.cc/rspjpmcV/save-2-NTSC.png](https://i.postimg.cc/rspjpmcV/save-2-NTSC.png)

Yes I can use (load\save) save game for PAL and NTSC games.
Previously I have uploaded save from 2 different games from 2 different regions (NTSC\PAL):
https://github.com/aldostools/IRISMAN/issues/12#issuecomment-643686946.

**Comment by aldostools** [comment]
FYI The memory save must be assigned on XMB (prior launch the game).

**Comment by J013k** [comment]
Memory Card(s) seems to work with webMAN MOD 1.47.28 and with ManaGunZ v1.38.

I even downloaded previous version of IRISMAN 4.85, but it didn't help.

**Comment by Markamiga500** [comment]
in order to be able to create save files in a PAL (F / france) game you
need to be able to start it
....I start my PAL games with the homebrew (available at Brewlologye) :
COBRA USB psx launcher.

its the ONLY way i found to be able to launch/start my PAL games on a NTSC
console
otherwise with other mean (webman mod) i alway got the Sony error code
80028F10

???

**Comment by J013k** [comment]
Can you post **some pictures** how you make game save?

Translated:
Pouvez-vous **poster des photos** de la façon dont vous sauvegardez le gibier ?

**Comment by Markamiga500** [comment]
I told you Im not able to create save files , now you ask me how i create
save files

I can MOUNT , LAUNCH & CREATE SAVE FILES for psx ntsc iso
I can MOUNT psx PAL iso with webman , I HAVE to start them using COBRA USB
PSX LAUNCHER (Brewology) ....therefore , launching theses PAL iso with
Cobra usb psx launcher DO NOT ALLOW ME to create save files.

its the ONLY way , the ONLY Solution i found so far to launch PAL iso on a
NTSC Console/Television set (Samsung HD tv)
.....but that setting DONT ALLOW ME TO CREATE SAVE FILES with all theses
PAL iso games...its frustrating...

**Comment by J013k** [comment]
Could this problem be somehow related to FW?
I mean I've checked quite old version 4.84.3 and I've had the same problem with VMC.

I remember that everything was fine (until ver 4.86) with old builds when I was on Rebug 4.85.1 LITE.

Unfortunately newer builds weren't working with Rebug 4.85.1 LITE...
so I needed to update to Rebug 4.86.1 LITE:
https://github.com/aldostools/IRISMAN/issues/5#issuecomment-616022915.
